src/main/resources/hudson/scm/subversion/Messages.properties

```diff
@@ -90,4 +90,4 @@ CredentialsSVNAuthenticationProviderImpl.credentials_in_realm=Found credentials
 CredentialsSVNAuthenticationProviderImpl.sole_credentials=Using sole credentials {0} in realm \u2018{1}\u2019
 CredentialsSVNAuthenticationProviderImpl.missing_credentials=No credentials found for realm \u2018{0}\u2019 among \u2018{1}\u2019; falling back to {2}
 
-
+SubversionSCM.browsers.VisualSVN=VisualSVN
```

Comment:
Best to add a trailing newline.

Reply:
I just merged with latest changes and added trailing newline to the messages.properties file.

However I'm not sure about the request to change the param from URL to String. Almost every implementation (except Assembla) has it in the same way (as URL). Therefore I propose to do not change the parameter type, since I'm not able to verify the correctness of the change anymore. If it will be problem for Snippet Generator then someone will have to change it in all others implementations anyway.

Please, can we finish it now? We are discussing PR for 2 years now.

Reply:
Unfortunately it's what we have for barely maintained plugins.
I and @jglick spent some time on cleaning up the PR queue this summer, but there were other maintainers before.
@kuisathaverat was interested in taking ownership of the plugin tho

Reply:
@arnosthavelka I made a PR with the suggested pending changes, merge it on your branch and I think that this PR is ready to merge

https://github.com/arnosthavelka/subversion-plugin/pull/1